index.html

```diff
@@ -36,7 +36,7 @@ <h1>Submission Page</h1>
 			<div class="column submit-column">
```

Comment:
This file needs to be deleted.

Reply:
can I change in next PR, I will be making some changes in it. like using it as a homepage.

Reply:
You can but it's a bad thing.

- It does not look like you forked this branch from master (probably from your existing branch)
- Those commits will end up being duplicated in the master when it is finally merged.

Best thing would be that you fork a new clean branch, code, and submit that in a new PR after closing this one.

Reply:
ok closing this one. this time one issue at a time